doc/mkdocs/docs/api/basic_json/operator_spaceship.md

```diff
@@ -0,0 +1,70 @@
+# <small>nlohmann::basic_json::</small>operator<=>
+
+```cpp
+// since C++20
+class basic_json {
```

Comment:
I'm obviously not putting down the complete template definition here but in other places I've abbreviated the template with `template<...>`.

Should I simply leave it off everywhere? It should be clear enough from context that `basic_json` refers to the main template class.

Reply:
In the other API pages, we do not repeat the class `basic_json` - it's only hinted in the title. I just copied this way from cppreference.com - or do I misunderstand?

Reply:
Well, previously operators where non-member friend functions. In C++20 they become member functions. This particular page (about `operator<=>`) doesn't exist prior to C++20 so it's not needed here. But the problem still exists on the other operator pages.

Reply:
cppreference is doing it like this:

- member function: https://en.cppreference.com/w/cpp/string/basic_string/operator%2B%3D

- nonmember function: https://en.cppreference.com/w/cpp/string/basic_string/operator%2B

Reply:
But we have them on the same page. Going from non-member to member. That's why I added the `class basic_json { }` explicitly.

Reply:
I see. Let's keep it this way until we find a better way. I like how cppreference is a bit more colorful...